contract-ui/tabs/embed/components/embed-setup.tsx

```diff
@@ -30,6 +30,21 @@ import {
   Heading,
 } from "tw-components";
 
+export function minimizeChain(chain: Chain) {
```

Comment:
let's instead change this in the package please

Reply:
Moved it to dashboard because that function is only used in dashboard

Reply:
Done